src/steps/check-lead-activity.ts

```diff
@@ -118,14 +144,14 @@ export class CheckLeadActivityStep extends BaseStep implements StepInterface {
           expectedAttributes.map(attr => `${attr.name} = ${attr.value}`).join(', '),
           stepData.activityTypeIdOrName,
           attributes.map(attr => JSON.stringify(attr, null, 2)).join('\n\n'),
-        ]);
+        ],               [activityRecords]);
```

Comment:
- [ ] Nit: The huge amount of whitespace on some of these are...  Odd.

Reply:
Ah yes. `tslint`

Reply:
😐